templates/designatebackendbind9/config-named/rndc.conf

```diff
@@ -0,0 +1,7 @@
+// The current PODs bind9 address is set by the initcontainer.
+// WARNING: if the required information is not available
+// the bind service will be configured to listen to all IPv4
+// addresses
+controls {
+        inet * port 953 allow { {{.AllowCIDR}}; } keys { "rndc-key"; };
```

Comment:
Maybe we should have a comment like
TODO: The '*'s need to be replaced by actual addresses.
after inet
(I see only one address in my named.conf after inet)